packages/pigeon/lib/objc_generator.dart

```diff
@@ -487,7 +487,7 @@ void _writeHostApiSource(Indent indent, ObjcOptions options, Api api) {
               func.isAsynchronous ? 'completion' : 'error';
           final String selector = _getSelector(func, lastSelectorComponent);
           indent.writeln(
-              'NSCAssert([api respondsToSelector:@selector($selector)], @"$apiName api doesn\'t respond to @selector($selector)");');
+              'NSCAssert([api respondsToSelector:@selector($selector)], @"$apiName api (%@) doesn\'t respond to @selector($selector)", api);');
```

Comment:
I would just remove "api" from the string. If you're reading the log message rather than looking at the code it's meaningless (nobody who hits this is going to know what Pigeon calls a variable in auto-generated code), and if you are reading the code the variable name is already right there in the line.

I can't think of any benefit to having it, and I (who *had* seen this code already, which is going to be very rare) found it confusing when I was reading the log message in the bug. I couldn't understand what it meant until I came and looked at the code.

Reply:
Depending on the name of their api, it's helpful to know we are talking about a HostApi or FlutterApi, denoted as "api" here.

Reply:
The string "api" in a log message does not help the reader distinguish between HostApi and FlutterApi; they are both APIs, and both have "Api" in the name.